docs/anchoring-assets/anchor-platform/sep24/setting-up-production-server.mdx

```diff
@@ -1,19 +1,40 @@
 ---
 title: Set Up a Production Server
-sidebar_position: 40
+sidebar_position: 60
 ---
 
+import { CodeExample } from "@site/src/components/CodeExample";
+
 Once the test server is live and you have tested both deposit and withdraw flows, it's time to get started with the real deploy connected to real KYC and real banking rails providers. Before using any banking APIs, it's critical that you perform a full security audit on the system to make sure that there aren't any vulnerabilities.
 
 ## Deploying a Secure Environment
 
 Make sure to keep the test server up, and deploy the production (mainnet) system in a separate environment. Having two deploys allows you to validate new features on the testnet before moving them to the final production deploy. You can also have a third staging environment if there's a big team working on this codebase and/or there will be many pushes to be tested internally before sharing with other institutions.
 
-To switch to Stellar's public (mainnet) network, all you have to do is change the network [passphrase](../../../encyclopedia/network-passphrases.mdx) (for authenticating requests) and [Horizon URL](https://horizon.stellar.org/). Stellar SDKs have all the built-in logic to switch from test to public with minor changes to the codebase.
+To switch to Stellar's public (mainnet) network, all you have to do is change the network [passphrase](../../../encyclopedia/network-passphrases.mdx) (for authenticating requests) and [Horizon URL](https://horizon.stellar.org/).
+
+First, you need to change your info file (`dev.stellar.toml`):
+
+<CodeExample>
+
+```toml
+# dev.stellar.toml
+NETWORK_PASSPHRASE = "Public Global Stellar Network ; September 2015"
+```
```

Comment:
I would encourage readers to create different files for their production configuration by just copying existing files and renaming them to remove the `dev.` prefix. They'll then have to update any references to the files.